packages/daimo-common/src/foreignCoin.ts

```diff
@@ -66,3 +95,18 @@ export function isAmountDust(
 
   return true;
 }
+
+// Get home coin by address (defaults to daimoUSDC).
+export function getHomeCoinByAddress(
+  address: string,
+  chainId?: number
+): ForeignCoin {
+  return allHomeCoins.get(address) ?? daimoUSDC;
```

Comment:
defensive coding = ensure valid inputs, throw error otherwise

- `address: Address`
- remove `chainId` if not used yet (or assert `chainId === 8453`)
- i'd throw an error if address is not a supported home coin